src/keyszer/output.py

```diff
@@ -10,6 +10,21 @@
 
 VIRT_DEVICE_PREFIX = "Keyszer (virtual)"
 
+# keystroke delays to work around modifier timing problems
+key_pre_delay_ms = 0
+key_post_delay_ms = 0
+
+
+def set_keystroke_delay(keystroke_delay=0):
+    """Used from config_api to pass keystroke delay from user's config."""
+    _kd = keystroke_delay
+    global key_pre_delay_ms
+    global key_post_delay_ms
+    key_pre_delay_ms = round( (_kd / 1500), 3) if _kd <= 200 and _kd > 0 else 0
+    key_post_delay_ms = round( (_kd / 1000), 3) if _kd <= 200 and _kd > 0 else 0
```

Comment:
What is all this logic?  Shouldn't the API be making sure we already have sane values?

Reply:
Yes, I see what you mean. All of that should be done in the original API function. I'll fix it.